store.go

```diff
@@ -280,18 +280,50 @@ func Move(ctx context.Context, s Store, src, des Object) error {
 		}
 	}
 
+	//	// Slow path, copy locally then up to des
```

Comment:
I'm experimenting with doing the move with out using a local intermediate file, TODO remove this if not used before merging the PR.